tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -90,6 +90,8 @@ def import_gluster(parameters):
     ]['tags']['tendrl-gluster-integration']
     config_data = {"etcd_port": int(NS.config.data['etcd_port']),
                    "etcd_connection": str(NS.config.data['etcd_connection']),
+                   "etcd_username": "root",
+                   "etcd_password": str(NS.config.data['etcd_password']),
```

Comment:
Check if etcd_username and etcd_password actually exist in config before setting them. Don't set them if they don't exist (in case of cert file Based with)

Reply:
Ah yes. will change the code accordingly.